lib/client.rb

```diff
@@ -68,11 +76,12 @@ def self.tls_using_sentinel(credentials, version='')
           verify_mode: OpenSSL::SSL::VERIFY_NONE
         }
       end
-
+      @sentinel_client ||= SentinelClient.new(redis_credentials)
+      master_info = @sentinel_client.get_master_address
```

Comment:
any specific reason to change prior implementation? if not then let's not change the prior implementation and pass `sentinels` array in `Redis.new` method

Reply:
Yes, it is changed since we need sentinel password which comes from Redis credentials. passing redis credentials will fetch values for host, port, master and sentinel_password.

Reply:
can we please change this to use `sentinel_password` with upgraded version of ruby gem as per the recent spike

Reply:
Yes, implemented the change